demisto_sdk/__main__.py

```diff
@@ -439,10 +439,15 @@ def zip_packs(**kwargs) -> int:
     "--allow-skipped",
     help="Don't fail on skipped integrations or when all test playbooks are skipped.",
     is_flag=True)
+@click.option(
+    "--without-multiprocessing",
```

Comment:
perhaps `single-process` or `no-multiprocessing`? not that important anyway

Reply:
done